docker/k8s/README.md

```diff
@@ -14,12 +14,16 @@ in SGX-SIM mode.
     ```bash
     kubectl create -f tcf-config-map.yaml
     ```
-2. Start the LMDB server
+2. Enable `remote_url` in [tcs_config.toml](../../config/tcs_config.toml)
```

Comment:
Instead of relative path, can we give full path - TCFHOME/config/tcs_config.toml?

Reply:
This is a hyperlink which anyone can click and traverse.

Reply:
Ok